app/models/defendant.rb

```diff
@@ -23,10 +23,6 @@ class Defendant < ApplicationRecord
   validates_with DefendantValidator
   validates_with DefendantSubModelValidator
 
-  acts_as_gov_uk_date :date_of_birth,
-                      validate_if: :validate_date?,
-                      error_clash_behaviour: :override_with_gov_uk_date_field_error
-
```

Comment:
This is related to the `defendants` form_step, is this required for `supporting_evidence`?